deploy-edgeX.sh

```diff
@@ -25,9 +25,11 @@
 #    . $(dirname "$0")/bin/env.sh
 #fi
 
+source ./sync.sh
```

Comment:
Suggest just execute
```
./sync.sh
```

and make sync.sh executable.

Reply:
fixed